app/src/main/kotlin/no/nav/tiltakspenger/vedtak/routes/behandling/benk/BehandlingBenkRoutes.kt

```diff
@@ -73,7 +75,8 @@ fun Route.behandlingBenkRoutes(
             call.withBody<BehandlingIdDTO> { body ->
                 val søknadId = SøknadId.fromString(body.id)
                 val correlationId = call.correlationId()
-                sakService.startFørstegangsbehandling(søknadId, saksbehandler, correlationId = correlationId).fold(
+                val sakId = søknadService.hentSakIdForSoknad(søknadId)
```

Comment:
Jeg synes frontend bør sende med både sakId+søknadId. Og helst flyttes til pathen /sak/{sakId}/soknad/{soknadId}/startbehandling f.eks.

Reply:
Hvorfor trenger den det egentlig..? Er det for å unngå å hente saken? Og, hvis sakId uansett blir en del av søknaden, forsvinner ikke problemet da? 

Ellers er jeg enig i at det kunne vært fint å endre endepuntket, men jeg vet ikke hvor lurt det er å gjøre det som en del av denne oppgaven som allerede er veldig stor. Da må jeg jo endre i frontenden også :/

Reply:
Enig, vi har dette i bakhodet til senere!